docs/api.rst

```diff
@@ -12464,7 +12464,10 @@ Clear Icons/Avatars Cache
 
 .. http:post:: /api/(version)/cache/<cache_type>/clear
 
-   Doing a POST on this endpoint will clear the cache of avatars/icons depending on the ``cache_type``. Valid options of ``cache_type`` are ``icons`` & ``avatars``.
+   Doing a POST on this endpoint will clear the cache of avatars/icons depending on the ``cache_type``. Valid options of ``cache_type`` are ``icons``, ``avatars``, ``curve``, ``yearn``, ``makerdao``. For curve, yearn and makerdao also requiries and updates the cache.
+
+   .. note::
+      This endpoint can also be queried asynchronously by using ``"async_query": true``.
```

Comment:
I would not mess with this endpoint and just create a new one since you are not really invalidating anything, just re-querying the the missing pools